src/main/java/org/jenkinsci/plugins/mesos/api/LaunchCommandBuilder.java

```diff
@@ -60,7 +60,7 @@ public LaunchCommandBuilder() {}
   private List<FetchUri> additionalFetchUris = Collections.emptyList();
   private Optional<ContainerInfo> containerInfo = Optional.empty();
   private AgentCommandStyle agentCommandStyle = AgentCommandStyle.Linux;
-  private DomainFilter domainInfoFilter = HomeRegionFilter$.MODULE$;
+  private DomainFilter domainInfoFilter = AnyDomain$.MODULE$;
```

Comment:
Maybe split `LaunchCommandBuilder` changes into a seperate PR focused on switching the default back to `AnyDomain`. Keeps the rest of this PR focused on networking changes, easier to cherry pick etc... in the future.